modules/shared/src/main/scala/org/tesselation/schema/cluster.scala

```diff
@@ -19,6 +19,11 @@ object cluster {
   @derive(decoder, encoder, show)
   case class PeerToJoin(id: PeerId, ip: Host, p2pPort: Port)
 
+  object PeerToJoin {
+    implicit def toPeer(peer: PeerToJoin): Peer =
+      Peer(peer.id, peer.ip, peer.p2pPort, peer.p2pPort)
```

Comment:
Is that correct? The same port used twice?

Reply:
Yes, at least for now.